src/clients/apiClient.ts

```diff
@@ -54,7 +54,7 @@ export interface WhaleSighting {
   dateOfSighting: string,
   locationLatitude: number,
   locationLongitude: number,
-  photoImageUrl: string,
+  photoImageURL: string,
```

Comment:
It is better practice to use camel case here so should be updated in the SightingDetail.tsx file
```suggestion
  photoImageUrl: string,
```